index.d.ts

```diff
@@ -3111,6 +3111,10 @@ export interface TooltipProps {
    * The appearance of the Tooltip.
    */
   appearance?: TooltipAppearance
+  /**
+   * The id of the tooltip.
+   */
+  id: string,
```

Comment:
`id` should be optional
```suggestion
  id?: string
```